build/publish/PublishDebian.targets

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <ItemGroup>
-    <SdkInstallerFileItemGroup Include="$(SdkInstallerFile)" />
-  </ItemGroup>
+  <PropertyGroup>
+    <SdkDebianPackageName>dotnet-dev-$(SdkVersion)</SdkDebianPackageName>
```

Comment:
It is used, that is the name of package name
for example the user should use

sudo apt-get install dotnet-dev-$(SdkVersion)

It was set in Packaging step. However, that prop cannot be shared with Publish.